src/archiefvernietigingscomponent/destruction/tests/test_destruction_list_create.py

```diff
@@ -93,3 +93,37 @@ def test_list_with_short_review_process(self):
             b'<script id="short-review-zaaktypes" type="application/json">["http://example.com/zaak/uuid-1"]</script>',
             response.content,
         )
+
+    def test_reminder(self):
```

Comment:
```suggestion
    def test_assigned_on_field_populated_on_assignment(self):
```

... or some better name! It helps to give more specific names to tests.